opencti-platform/opencti-front/src/private/components/analyses/groupings/GroupingKnowledgeGraph.jsx

```diff
@@ -93,6 +94,63 @@ const groupingKnowledgeGraphCheckRelationQuery = graphql`
   }
 `;
 
+const groupingKnowledgeGraphCheckObjectQuery = graphql`
+  query GroupingKnowledgeGraphCheckObjectQuery($id: String!) {
+    stixObjectOrStixRelationship(id: $id) {
+      ... on BasicObject {
+        id
+      }
+      ... on StixCoreObject {
+        is_inferred
+        parent_types
+        reports {
```

Comment:
We should get the groupings and not the reports

Reply:
good catch

Reply:
fixed